controllers/installation_controller.go

```diff
@@ -32,9 +35,9 @@ const (
 type InstallationReconciler struct {
 	client.Client
 	Log              logr.Logger
-	PorterGRPCClient PorterClient
 	Recorder         record.EventRecorder
 	Scheme           *runtime.Scheme
+	CreateGRPCClient func(ctx context.Context) (porterv1alpha1.PorterClient, ClientConn, error)
```

Comment:
Needed as we moved this to the parent function when doing outputs in the reconciler. This method can be overridden when testing and the interfaces are all mocked to show code path.